kalohq/ui
# Removed outdated flow

Removed flow!

Converted all `types` to `PropTypes`.

### Next step
Reenable the eslint checks about `PropTypes` and add / fix them :)

#### How to verify
1. Check out this branch
1. `npm install`
1. `cd site`
1. `npm install`
1. `npm start`
1. Check that everything looks fine in the style guides
